### PR TITLE
documentation: usage example updates

### PR DESCRIPTION
**Pull request by gr2m**
This will look much better once the code is on the side

-----
[View rendered docs/src/pages/api/00_usage.md](https://github.com/octokit/rest.js/blob/docs/usage-example/docs/src/pages/api/00_usage.md)
[View rendered docs/src/pages/api/03_request_formats.md](https://github.com/octokit/rest.js/blob/docs/usage-example/docs/src/pages/api/03_request_formats.md)

### CODE REVIEW

File: docs/src/pages/api/00_usage.md [hunk 1]
```diff
@@ -2,86 +2,197 @@
...
 
-### Node
+Import the Octokit constructor.
 
-Install with `npm install @octokit/rest`.
+For direct usage in browsers, download `octokit-rest.min.js` from the [latest release](https://github.com/octokit/rest.js/releases/latest) and import it using a `<script src="octokit-rest.min.js"></script>` tag.
 
 ```js
-const octokit = require('@octokit/rest')()
+const Octokit = require('@octokit/rest')
```
Comment (BarbourSmith) [inline docs/src/pages/api/00_usage.md:10]: Is it also possible to use the `import Octokit from 'octokit/rest'`? style of import?

File: docs/src/pages/api/00_usage.md [hunk 2]
```diff
@@ -2,86 +2,197 @@
 title: "Usage"
 ---
 
-### Node
+Import the Octokit constructor.
 
-Install with `npm install @octokit/rest`.
+For direct usage in browsers, download `octokit-rest.min.js` from the [latest release](https://github.com/octokit/rest.js/releases/latest) and import it using a `<script src="octokit-rest.min.js"></script>` tag.
```
Comment (BarbourSmith) [inline docs/src/pages/api/00_usage.md:7]: Why the change in focus from using npm to using the download in browsers?

Comment (gr2m) [reply docs/src/pages/api/00_usage.md:7]: there is no change of focus, it’s a universal library that should run in all environments. I just updated the documentation with the content from the README.md

File: docs/src/pages/api/00_usage.md (hunk 1 above)
Comment (eine) [reply docs/src/pages/api/00_usage.md:10]: I'm also interested on this.
